# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1366232064 14400
#      Wed Apr 17 16:54:24 2013 -0400
# Node ID 52c66133415471ea029b11a3bafd0371e694d5fc
# Parent  67d4111d078db76ffc7b92758971e7e3add8fc93
avoid trouble with @sc{..} in section heading/node name

* java.txi, octave.texi: Use Matlab instead of @sc{matlab} in section
heading, node name, and corresponding menu item.

diff --git a/doc/interpreter/java.txi b/doc/interpreter/java.txi
--- a/doc/interpreter/java.txi
+++ b/doc/interpreter/java.txi
@@ -136,26 +136,26 @@ of dialog box.
 @cindex dialog, displaying a warning dialog
 @DOCSTRING(warndlg)
 
 @c ------------------------------------------------------------------------
 @node FAQ - Frequently asked Questions
 @section FAQ - Frequently asked Questions
 
 @menu
-* How to distinguish between Octave and @sc{matlab}?::
+* How to distinguish between Octave and Matlab?::
 * How to make Java classes available?::
 * How to create an instance of a Java class?::
 * How can I handle memory limitations?::
 * Which @TeX{} symbols are implemented in the dialog functions?::
 @end menu
 
 @c ------------------------------------------------------------------------
-@node How to distinguish between Octave and @sc{matlab}?
-@subsection How to distinguish between Octave and @sc{matlab}?
+@node How to distinguish between Octave and Matlab?
+@subsection How to distinguish between Octave and Matlab?
 @anchor{doc-FAQ}
 @c - index -
 @cindex Octave and @sc{matlab}, how to distinguish between
 @c - index -
 
 Octave and @sc{matlab} are very similar, but handle Java slightly different.
 Therefore it may be necessary to detect the environment and use the appropriate
 functions.  The following function can be used to detect the environment.  Due
diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -766,17 +766,17 @@ Networking Utilities
 Java Interface
 
 * Java Interface Functions::
 * Dialog Box Functions::
 * FAQ - Frequently asked Questions::
 
 FAQ - Frequently asked Questions
 
-* How to distinguish between Octave and @sc{matlab}?::
+* How to distinguish between Octave and Matlab?::
 * How to make Java classes available?::
 * How to create an instance of a Java class?::
 * How can I handle memory limitations?::
 * Which @TeX{} symbols are implemented in the dialog functions?::
 
 Packages
 
 * Installing and Removing Packages::  
